package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^3.9.2",
     "@material-ui/icons": "^3.0.2",
+    "casual-browserify": "^1.5.19-2",
```

Comment:
we don't need the Browserify version - replace with this one https://www.npmjs.com/package/casual - otherwise looks good!